src/history.ts

```diff
@@ -11,12 +10,11 @@ export interface HistoryEntry {
 
 interface DehydratedExcalidrawElement {
   id: string;
-  version: number;
   versionNonce: number;
 }
 
 interface DehydratedHistoryEntry {
-  appState: ReturnType<typeof clearAppStatePropertiesForHistory>;
+  appState: string;
```

Comment:
not a huge fan of this change -- i think that doing a deep copy / deep equal might be faster and have cheaper memory usage. but this is so small that it doesn't matter so i'm fine with this change.